invenio_drafts_resources/services/records/service.py

```diff
@@ -65,6 +66,7 @@ def search_drafts(self, identity, params=None, links_config=None,
             links_config=links_config,
             es_preference=es_preference,
             record_cls=self.draft_cls,
+            extra_filter=Q('term', **{'has_draft': False})
```

Comment:
Comment: You can do ``Q('term', has_draft=False)``. The other syntax is just necessary when you have e.g. dots.